lib/authentication.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const MAX_PASS_LENGTH = 128;
+const MIN_PASS_LENGTH = 10;
+const MIN_PHRASE_LENGTH = 20;
+
+const MIN_LOGIN_LENGTH = 6;
+const MAX_LOGIN_LENGTH = 255;
+
+const passwordTests = {
```

Comment:
These don't seem to be exported, is there a way to pass wanted tests to the appropriate functions?

Reply:
Yes. You just need to pass the names of the necessary tests to the test function.

Reply:
Maybe that's a place to use our new `Enum` class? @tshemsedinov what do you think?

Reply:
agree :+1:

Reply:
Please add here example with enums to compare